### PR TITLE
tabby and loner names

### PR DESCRIPTION
**Pull request by selkirks**
added loner names + the swirl suffix to name.json. from the name doc and the name thread

### CODE REVIEW

**Review by Lixxis — APPROVED** [review]
The only thing I'm not sure about it are the tortie_pelt_suffixes renaming, because I don't interact with spirits much.
@sedgestripe @scribblecrumb maybe one of you can take a look?

File: resources/dicts/names/names.json [hunk 1]
```diff
@@ -297,115 +296,114 @@
...
     "Mr. Anderson", "Mr. Kitty Whiskers", "Mr. Kitty", "Mr. Midnight", "Mr. Mistoffolees", "Mr. Mustard", "Mr. Oreo", "Mr. President", "Mr. Princess", "Mr. Sir", "Mr. Whiskers", "Mr. Wigglebottom", 
     "Mucha", "Mudkip", "Mufasa", "Muffet", "Muffy", "Mullido", "Mumbai", "Mungojerrie", "Munkustrap", "Murder", "Mushroom", "Mustard", "Myko", "Nabi", 
-    "Nadia", "Nadir", "Nadja", "Nagi", "Nagito", "Nakala", "Nakeena", "Nala", "Namielle", "Namu", "Nandor", "Naomi", "Nari", "Neapolitan", "Neel", "Nefertiti", "Neil", "Nemo", "Neo", "Nephrite", "Neptune", "Ness", "Nessie", "Neuron", "Neutron", "Nevaeh", 
-    "Neytiri", "Nezuko", "Nibbles", "Nick", "Nidorina", "Nightcat", "Nightmare", "Nikki", "Nile", "Niles", "Nimbus", "Nine", "Ninetales", "Ninja", "Nintendo", "Nintendo DS", "Nintendo Switch", "Nirav", 
+    "Nadia", "Nadir", "Nadja", "Nagi", "Nagito", "Nakala", "Nakeena", "Nala", "Namielle", "Namu", "Nandor", "Naomi", "Nari", "Naughty Wilbur", "Neapolitan", "Neel", "Nefertiti", "Neil", "Nemo", "Neo", "Nephrite", "Neptune", "Ness", "Nessie", "Neuron", "Neutron", "Nevaeh",
```
Comment (keyraven) [inline resources/dicts/names/names.json:370]: As much as I love "Naughty Wilbur", it seems strange to add a normal name with an adjective in front of it. Maybe just add "Wilbur"?

**Review by keyraven — COMMENTED** [review]
Looks good! Just one comment

**Comment by scribblecrumb** [comment]
> The only thing I'm not sure about it are the tortie_pelt_suffixes renaming, because I don't interact with spirits much. @sedgestripe @scribblecrumb maybe one of you can take a look?

Not quite my area of expertise, but I know @Ryos00 has done sprite work

**Comment by selkirks** [comment]
> > The only thing I'm not sure about it are the tortie_pelt_suffixes renaming, because I don't interact with spirits much. @sedgestripe @scribblecrumb maybe one of you can take a look?
> 
> 
> 
> Not quite my area of expertise, but I know @Ryos00 has done sprite work

i think its from? https://github.com/ClanGenOfficial/clangen/pull/2163
i didnt touch that section aside from adding swirl as a suffix to somr

**Comment by selkirks** [comment]
my PR is trying to reverse the changes made by 2163 so ill close this till im home and fix it on my end

**Pull request closed without merging**